File: src/main/java/club/xiaojiawei/bean/area/RemovedfromgameArea.java
package club.xiaojiawei.bean.area;

import club.xiaojiawei.bean.Card;

/**
 * 移除区
 * @author 肖嘉威
 * @date 2022/12/3 21:37
 */
public class RemovedfromgameArea extends Area{
    public RemovedfromgameArea() {
        super(Integer.MAX_VALUE);
    }
    @Override
    protected void addZeroCard(Card card) {
        add(card);
    }
}

File: hs-script-card-sdk/src/main/kotlin/club/xiaojiawei/bean/area/RemovedfromgameArea.kt
package club.xiaojiawei.bean.area

import club.xiaojiawei.bean.Card
import club.xiaojiawei.bean.Player

/**
 * 移除区
 *
 * @author 肖嘉威
 * @date 2022/12/3 21:37
 */
class RemovedfromgameArea(allowLog: Boolean = false, player: Player) :
    Area(allowLog = allowLog, maxSize = Int.MAX_VALUE, player = player) {

    override fun addZeroCard(card: Card?) {
        add(card)
    }

}
